### DISCUSSION TITLE
Tooltip autoplacement missing

### DISCUSSION THREAD

**Discussion by alyrik** [heroui-inc/heroui#421] [Feedback]
Hi! Thank you for beautiful library!

It turned out that Tooltip component has only static placement which often results in overflowing the screen:  

![Screenshot 2022-04-21 at 18 28 09](https://user-images.githubusercontent.com/10388022/164507371-85ce4f84-3dfd-4446-ad5e-645b8c3d60f7.png)

Is it possible to avoid this problem? Are there any plans to introduce autoplacement feature for Tooltip?

**Comment by jrgarciadev** [comment]
Hey @alyrik thank you for using it 🙌🏻 

We will release a Popover component https://github.com/nextui-org/nextui/pull/420 that is automatically adjusted when is likely to be offscreen or when the window is resized. Then we will use it as a base for the Tooltip component

**Reply by alyrik** [reply]
That's amazing! Thank you!
